test/_e2e/trigger_http_test.go

```diff
@@ -86,7 +86,7 @@ var defaultFunctionsHttpValidators = []FunctionHttpResponsivenessValidator{
 		expects:   `{"message":"hello"}`,
 	},
 	{runtime: "springboot",
-		targetUrl: "%s/uppercase",
+		targetUrl: "%s/echo",
 		method:    "POST",
 		bodyData:  "message=hello",
 		expects:   `MESSAGE=HELLO`,
```

Comment:
Shouldn't be this lowercase?

Reply:
Thank you for your help @matejvasek.

I changed the `/uppercase` endpoint to `/echo`.

Reply:
I got it, e2e_tests looks for https://github.com/mcruzdev/kn-plugin-func/blob/mcruzdev-1062-echo/test/_e2e/update_templates/springboot/http/src/main/java/functions/CloudFunctionApplication.java file.

Reply:
I will change the expected result, thank you.

Reply:
expects:   `MESSAGE=HELLO`,
should be:
		expects:   `message=hello`,

Reply:
> I got it, e2e_tests looks for https://github.com/mcruzdev/kn-plugin-func/blob/mcruzdev-1062-echo/test/_e2e/update_templates/springboot/http/src/main/java/functions/CloudFunctionApplication.java file.

thats cloudevents thing not plain http thing